docs/src/extend/custom-parsers.md

```diff
@@ -74,8 +80,72 @@ interface Token {
 
 The range indexes of all tokens and comments must not overlap with the range of other tokens and comments.
 
-### The `Literal` node:
+### The `Literal` Node
 
 The `Literal` node must have `raw` property.
 
 * `raw` (`string`) is the source code of this literal. This is the same as `code.slice(node.range[0], node.range[1])`.
+
+## Packaging a Custom Parser
+
+To publish your custom parser to npm, perform the following:
+
+1. Create a custom parser following the [Creating a Custom Parser](#creating-a-custom-parser) section above.
+1. Create an npm package for the custom parser.
+1. In your `package.json` file, set the [`main`](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#main) field as the file that exports your custom parser.
+1. Publish the npm package.
```

Comment:
I won't nitpick on this, but I'm afraid that _"Create an npm package"_ and _"Publish the npm package"_ are not self-explanatory. Maybe we could like these steps to [`npm init`](https://docs.npmjs.com/cli/v9/commands/npm-init?v=true) and [`npm publish`](https://docs.npmjs.com/cli/v9/commands/npm-publish?v=true) respectively, or just show the commands to enter in the shell?

Reply:
I think the wording is okay, but would be helpful to link out to some supporting documentation.